app/src/ui/lib/authentication-form.tsx

```diff
@@ -72,19 +72,18 @@ export class AuthenticationForm extends React.Component<
   }
 
   public render() {
+    const content = this.props.supportsBasicAuth
+      ? this.renderEndpointRequiresWebFlow()
+      : this.renderSignInForm()
```

Comment:
Shouldn't this be the opposite? (show the message requiring web flow auth when the endpoint doesn't support basic auth).

(I think I introduced this bug in https://github.com/desktop/desktop/pull/9663/files#r419732770 😅 🤡 ). 

```suggestion
    const content = this.props.supportsBasicAuth
      ? this.renderSignInForm()
      : this.renderEndpointRequiresWebFlow()
```

Reply:
fixed in ac2e8c4!